components/reusable-components/navigation/navigation.module.scss

```diff
@@ -0,0 +1,62 @@
+.navbar {
+  background-color: var(--bg-color);
+}
+
+.navbarBrand {
+  color: #ffff;
+  font-size: 36px;
+  margin-left: 20px;
+  width: 227px;
+  height: 44px;
+  top: 67px;
+  left: 85px;
+}
+
+.navLink1 {
+  color: #ffff;
+  font-size: 25px;
+  margin-right: -370px;
+  margin-left: 430px;
+}
+
+.horizontalNav {
+  display: flex;
+  list-style: none;
+  padding: 10px;
+}
+
+.navContainer {
+  display: flex;
+  justify-content: space-between;
+}
+.leftContent {
+  display: flex;
+  align-items: center;
+}
+
+.rightContent {
+  display: flex;
+  align-items: center;
+}
+
+.customButton1 {
```

Comment:
I am not sure if you already covered the naming, but this class name is not descriptive enough. What does the 1 stand for?